app/views/headerFooterTemplate/Layout.scala.html

```diff
@@ -17,13 +17,15 @@
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @import uk.gov.hmrc.govukfrontend.views.viewmodels.header.Header
 @import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcStandardFooter
+@import views.html.templates.helpers.Link
 
 @this(
         govukLayout: GovukLayout,
         govukHeader: GovukHeader,
         head: Head,
         beforeBodyContent: BeforeBodyContent,
-        hmrcStandardFooter: HmrcStandardFooter
+        hmrcStandardFooter: HmrcStandardFooter,
+        link: Link,
```

Comment:
Not sure if you want this to be named getHelpLink: Link incase we have to add links in the future so you know which link it is?

Reply:
Will make the change.